### PR TITLE
further speedups to repeated_nextto_painting env

### PR DESCRIPTION
**Pull request by ronuchit**

### CODE REVIEW

File: tests/approaches/test_oracle_approach.py [hunk 1]
```diff
@@ -603,27 +603,57 @@ def test_oracle_approach_repeated_nextto_painting():
...
+        "painting_num_objs_test": [1],
+        # NOTE: We found hff to make planning for this problem significantly
+        # faster, and thus use it here so that the tests run quickly!
         "sesame_task_planning_heuristic": "hff"
     })
-    # NOTE: We found hff to make planning for this problem significantly
-    # faster, and thus use it here so that the tests run quickly!
     env = RepeatedNextToPaintingEnv()
-    env.seed(3210)  # This random seed necessary to get full coverage.
+    env.seed(123)
```
Comment (tomsilver) [inline tests/approaches/test_oracle_approach.py:613]: this should be unnecessary right? seed is called in initialization, and 123 is the default

**Review by NishanthJKumar — APPROVED** [review]
LGTM! Thanks for the ultra-fast turnaround on this!

File: tests/approaches/test_oracle_approach.py [hunk 2]
```diff
@@ -603,27 +603,57 @@ def test_oracle_approach_repeated_nextto_painting():
...
     train_tasks = env.get_train_tasks()
     approach = OracleApproach(env.predicates, env.options, env.types,
                               env.action_space, train_tasks)
     assert not approach.is_learning_based
-    for train_task in train_tasks:
-        policy = approach.solve(train_task, timeout=25)
-        assert policy_solves_task(policy, train_task, env.simulate)
-
+    approach.seed(123)
```
[the remark this answers is not in the document]
Comment (ronuchit) [reply tests/approaches/test_oracle_approach.py:618]: ditto

File: tests/approaches/test_oracle_approach.py (hunk 1 above)
Comment (ronuchit) [reply tests/approaches/test_oracle_approach.py:613]: opened an issue #600 i can do this later

**Pull request merged**